tfx/experimental/pipeline_testing/base_stub_executor.py

```diff
@@ -72,8 +72,7 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
     for output_key, artifact_list in output_dict.items():
       for artifact in artifact_list:
         dest = artifact.uri
-        component_id = artifact.producer_component
-        src = os.path.join(self._test_data_dir, component_id, output_key)
+        src = os.path.join(self._test_data_dir, self._component_id, output_key)
```

Comment:
Don't we need to handle `path.steps[1].index` here? It looks like that there is another depth in the recorded path.

Reply:
Is this handled? I cannot find any changes around here...